templates/zerver/help/import-from-rocketchat.md

```diff
@@ -1,98 +1,127 @@
-the most common configuration, run the following commands:
+{!import-self-hosted-server-tips.md!}
 
-```
-cd /home/zulip/deployments/current
-./scripts/stop-server
-./manage.py convert_rocketchat_data /home/zulip/rocketchat --output /home/zulip/converted_rocketchat_data
-./manage.py import "" /home/zulip/converted_rocketchat_data
-./scripts/start-server
-```
+    ```
+    cd /home/zulip/deployments/current
+    ./scripts/stop-server
+    ./manage.py convert_rocketchat_data /tmp/rocketchat_data --token <token> --output /tmp/converted_rocketchat_data
+    ./manage.py import '' /tmp/converted_rocketchat_data
+    ./scripts/start-server
+    ```
 
-This could take a few seconds to several minutes to run, depending on how
-much data you're importing. The server stop/restart is only necessary
-when importing on a server with minimal RAM, where an OOM kill might
-otherwise occur.
+    Alternatively, to import into a custom subdomain, run:
 
-**Import options**
+    ```
+    cd /home/zulip/deployments/current
+    ./scripts/stop-server
+    ./manage.py convert_rocketchat_data /tmp/rocketchat_data --token <token> --output /tmp/converted_rocketchat_data
+    ./manage.py import <subdomain> /tmp/converted_rocketchat_data
+    ./scripts/start-server
+    ```
 
-The commands above create an imported organization on the root domain
-(`EXTERNAL_HOST`) of the Zulip installation. You can also import into a
-custom subdomain, e.g. if you already have an existing organization on the
-root domain. Replace the last line above with the following, after replacing
-`<subdomain>` with the desired subdomain.
+1. Follow [step 4](https://zulip.readthedocs.io/en/stable/production/install.html#step-4-configure-and-use)
+   of the guide for [installing a new Zulip
+   server](https://zulip.readthedocs.io/en/stable/production/install.html).
 
-```
-./manage.py import <subdomain> /home/zulip/converted_rocketchat_data
-```
+{end_tabs}
 
-{!import-login.md!}
+#### Import details
 
-[upgrade-zulip-from-git]: https://zulip.readthedocs.io/en/latest/production/upgrade-or-modify.html#upgrading-from-a-git-repository
+Whether you are using Zulip Cloud or self-hosting Zulip, here are a few notes to
+keep in mind about the import process:
 
-## Caveats
+- Rocket.Chat does not export workspace settings, so you will need to [configure
+  the settings for your Zulip organization](/help/customize-organization-settings).
+  This includes settings like [email
+  visibility](/help/restrict-visibility-of-email-addresses),
+  [message editing permissions](/help/configure-message-editing-and-deletion#configure-message-editing-and-deletion_1),
+  and [how users can join your organization](/help/restrict-account-creation).
 
-This import tool is currently beta has the following known limitations:
+- Rocket.Chat does not export user settings, so users in your organization may
+  want to [customize their account settings](/help/getting-started-with-zulip).
```

Comment:
Is this documented somewhere officially by them, or just a thing we've noticed in practice when doing our imports?

Reply:
I think it makes sense to include this because Rocket.Chat doesn't have an official export tool, but it's also good to mention it as a reminder for users to check their account settings.